### PR TITLE
Minor cleanup and documentation tweaks

### PR DESCRIPTION
**Pull request by sitsofe**
- Fix up some broken formatting introduced with 074f08170af15ca8ebd469c3a1fa09c38f29904e ("doc: rewrite write_*_log sections")
- Reword buffer_compress_percentage, buffer_compress_chunk and dedupe_percentage sections.

### CODE REVIEW

File: init.c [hunk 1]
```diff
@@ -772,9 +772,6 @@ static int fixup_options(struct thread_data *td)
 			ret = warnings_fatal;
 		}
 
-		if (!fio_option_is_set(o, refill_buffers))
```
Comment (axboe) [inline init.c]: What am I missing here, I don't see where else it's being set for this?

Comment (sitsofe) [reply init.c]: I think it gets set over in `init_flags()` https://github.com/sitsofe/fio/blob/9164932ecbb3aa4b1538080f97452a2fdb3f02f1/init.c#L1112 and `init_flags()` is called before `fixup_options()` over in `add_job()` (see https://github.com/sitsofe/fio/blob/9164932ecbb3aa4b1538080f97452a2fdb3f02f1/init.c#L1359 ).

Comment (axboe) [reply init.c]: init_flags() may set the REFILL_BUFFERS flag, but not o->refill_buffers.

Comment (sitsofe) [reply init.c]: Ah you're right (e.g. in any case where you're not doing `compress_percentage`). OK let me drop that commit.

**Comment by sitsofe** [comment]
OK updated.

**Comment by sitsofe** [comment]
Hmm I wonder, should `if (!fio_option_is_set(o, refill_buffers))` needs to have `TD_F_REFILL_BUFFERS` inside the braces too...

**Comment by axboe** [comment]
I think that init_flags() should just be moved after we've fiddled with the options.

**Comment by axboe** [comment]
In any case, I've pulled this branch, since you dropped the bad commit.

**Pull request merged**